src/compute.ts

```diff
@@ -0,0 +1,183 @@
+import semver from 'semver'
+import { isNullString } from './utils'
+
+export enum Scheme {
+  Continuous = 'continuous',
+  Semantic = 'semantic',
+}
+
+function initialTag({
+  prefix,
+  suffix,
+  versionType,
+  tag,
+}: {
+  prefix?: string
+  suffix: string
+  versionType: semver.ReleaseType
+  tag: string
+}): string {
+  const isPrerelease = versionType === 'prerelease'
```

Comment:
Why did you drop the `Semantic` enum? Now we're string-comparing the release type in lots of places.